test/controllers/recipes_controller_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe RecipesControllerController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
```

Comment:
You need some controller tests! Here is my list of what I'm looking for:

Index:
- Valid search term
- Search term that produces no results (e.g. `adfohdaglhjalj`)
- No search term

Show:
- Valid URI
- Bogus URI
- No URI

Root
- Does it work at all (only one test)